lib/src/list.dart

```diff
@@ -326,3 +331,18 @@ class ListNotificationsController<T extends Object?> extends NotificationsContro
     streamController.addError(error);
   }
 }
+
+extension ListExtension<T> on List<T> {
+  void move(int from, int to) {
```

Comment:
Missing API doc. Looks strange in the documentation.

Reply:
Will add